### DISCUSSION TITLE
Terminology: 'Protocol' vs. 'Binding' vs. 'Solution' vs. 'Interaction Sequence'

### DISCUSSION THREAD

**Discussion by sebbader** [International-Data-Spaces-Association/IDS-G#71] [General]
To enable an efficient discussion (and writing), we need to agree on the right words. Please put your points in here, and then agree on what we actually mean when talking about each of the above terms.

Target: Add the *right* definitions to the [IDS-G Glossary](https://github.com/International-Data-Spaces-Association/IDS-G/tree/main/Glossary)

**Comment by sebbader-sap** [comment]
My view:
(Internet communication) `protocol`: [A communication protocol is a system of rules that allows two or more entities of a communications system to transmit information via any kind of variation of a physical quantity. The protocol defines the rules, syntax, semantics and synchronization of communication and possible error recovery methods. Protocols may be implemented by hardware, software, or a combination of both.](https://en.wikipedia.org/wiki/Communication_protocol) In particular however, I see HTTP, TCP, UDP, MQTT etc. as protocols, in contrast to business workflows.

`Transport protocol`: TCP/IP and other protocols on the same or close OSI layers.

`Binding` is used for instance in the [broker readme](https://github.com/International-Data-Spaces-Association/IDS-G/Communication/sequence-diagrams/data-connector-to-metadata-broker/README.md) or [IDS REST](https://github.com/International-Data-Spaces-Association/IDS-G/Communication/protocols/ids-rest/README.md) describes the details of an `interaction sequence` for an application layer protocol (like HTTP). Examples could be [OData](https://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part1-protocol.html), the [FIWARE NGSI-LD definitions](https://ngsi-ld-tutorials.readthedocs.io/en/latest/ngsi-ld-operations.html), or the [AAS API Operations](https://www.plattform-i40.de/IP/Redaktion/EN/Downloads/Publikation/Details_of_the_Asset_Administration_Shell_Part2_V1.html).

`Interaction sequence`: Stateful, synchronous or asynchronous sequence of exchanged messages. Examples: OAuth message flow, IDS policy negotiation.

`Solution`: I know this term more in the direction of an implementation technology, e.g., 'my Java SpringBoot-based server is a solution following the IDS Metadata Broker specification'.
